### ISSUE TITLE
Non-integer comic volume not shown on series page

### ISSUE THREAD

**Issue by pssandhu** [Kareadita/Kavita#1606]
**Describe the bug**
A comic volume with a non-integer number (e.g. v2.5) is not shown on the series page. However, it is detected by kavita as the file appears in the search results but cannot be read. Finishing reading volume 2 takes you to volume 3.

**To Reproduce**
1. Have a comic series with integer volumes 01 to 10 as well as volume 2.5. There is one file per volume.
2. Scan comic library to add series to kavita
3. Go to series page
4. Volume 2.5 is not shown but volumes 01 to 10 are there

**Expected behavior**
Volume 2.5 should appear between volumes 2 and 3.

**Desktop (please complete the following information):**
 - OS: Docker
 - Browser: Firefox (but probably any)
 - Version: Kavita 0.6.0.0

**Additional context**
A different series with one file per issue has a 16.1 issue and that works fine.

**Comment by majora2007** [comment]
Can you share an image for me? I'm not understanding on the first read.

**Comment by pssandhu** [comment]
Series page (volumes tab looks the same):
![image](https://user-images.githubusercontent.com/11130930/197366308-0abfca6f-c093-4690-8667-65ee5e6a1832.png)

Search results for volume 2.5:
![image](https://user-images.githubusercontent.com/11130930/197366318-d7842a5f-b64d-4276-b9ca-74c84cdeefb0.png)

File info for volume 2:
![image](https://user-images.githubusercontent.com/11130930/197366324-974d310e-ce18-4e52-9d17-0f44e9cded57.png)

Should have probably included these to begin with, sorry

**Comment by majora2007** [comment]
Okay, understood. I don't think partial volumes are actually supported, but I'll double check.

**Comment by majora2007** [comment]
I don't think so. Kavita only uses folder names for 2 things. 1 is we can't parse any details from the filename and 2 is when the file is a special, we use folder name as series.

I know that I coded it so volumes have a non float number for grouping. But I did make some changes in the last two releases. 

Volumes with floats for books do work but have special logic. I'll take a look but on holiday, so it will be a bit.

**Comment by pssandhu** [comment]
I have all the volumes in the same folder. Whether the folder has a number in the name or not makes no difference.

**Comment by majora2007** [comment]
Took a look, this is working as expected. I believe you likely have a ComicInfo in your 2.5 volume that is saying Volume is 2, hence Kavita is grouping. 

Please re-check for me. 

![image](https://user-images.githubusercontent.com/735851/201366402-49d8205d-0f82-43ba-8489-6eac1c235518.png)
```
Liar-Game v01.cbr
Liar-Game v1.5.cbr
```

**Comment by pssandhu** [comment]
I've moved the series to it's own library. If the library type is Comic it doesn't work but if the type is Manga it works as you've shown.

**Comment by majora2007** [comment]
First off, please update to v0.6.1. 6.0 has some security vulnerabilities on it that I patched, along with some other nice fixes.

As far as the issue goes, for Comic, you are correct. Partial volumes are not well supported in the Parser. I can reproduce and will enhance.
